src/lib/types.ts

```diff
@@ -166,12 +122,25 @@ export interface Session {
   expires_at?: number
   token_type: string
   user: User
-  aal_level?: string
-  amr?: AMREntry[]
 }
+
+/**
+ * An authentication methord reference (AMR) entry.
+ *
+ * An entry designates what method was used by the user to verify their
+ * identity and at what time.
+ *
+ * @see {@link GoTrueMFAApi#getAuthenticatorAssuranceLevel}.
+ */
 export interface AMREntry {
-  authentication_method: string
-  timestamp: string
+  /** Authentication method name. */
+  method: string // TODO: explicit method names / types
```

Comment:
@J0 Once we finalize the AMR methods, this needs to be changed to include them explicitly.